pipelines/foreman_split_puppet.yml

```diff
@@ -0,0 +1,109 @@
+- hosts: localhost
+  vars_files:
+    - vars/puppet_base.yml
+  roles:
+    - forklift
+
+- hosts:
+  - "{{ forklift_server_name }}"
+  - "{{ forklift_proxy_name }}"
+  - "{{ forklift_puppet_name }}"
+  become: yes
+  vars:
+    foreman_installer_skip_installer: true
+  vars_files:
+    - vars/puppet_base.yml
+  roles:
+    - role: forklift_versions
+      scenario: "{{ pipeline_type }}"
+      scenario_version: "{{ pipeline_version }}"
+    - role: foreman_server_repositories
+    - role: update_os_packages
+    - role: foreman_installer
+
+- hosts: "{{ forklift_puppet_name }}"
+  become: yes
+  vars_files:
+    - vars/puppet_base.yml
+  roles:
+    - role: forklift_versions
+      scenario: "{{ pipeline_type }}"
+      scenario_version: "{{ pipeline_version }}"
+    - role: foreman_installer
+      foreman_installer_options_internal_use_only:
+        - "--no-enable-foreman"
+        - "--no-enable-foreman-cli"
+        - "--no-enable-foreman-proxy"
```

Comment:
I wonder in these situations, would it be cleaner to add a puppet scenario to the installer?